beetmoverscript/task.py

```diff
@@ -22,26 +21,38 @@ def validate_task_schema(context):
     scriptworker.client.validate_json_schema(context.task, task_schema)
 
 
-def validate_task_scopes(context, manifest):
-    # make sure scopes exist and are properly set
-    scopes = context.task['scopes']
-    for scope in scopes:
-        if scope.startswith("project:releng:beetmover:"):
-            signing_cert_name = scope.split(':')[-1]
-            if re.search('^[0-9A-Za-z_-]+$', signing_cert_name) is not None:
-                break
-            log.warning('scope {} is malformed, skipping!'.format(scope))
-    else:
-        log.critical("No beetmover scopes!")
-        sys.exit(3)
-
-    # prevent scopes from low-security tree write in a release-type bucket
-    for k, v in RESTRICTED_BUCKET_PATHS.items():
-        for path in (manifest['s3_prefix_dated'], manifest['s3_prefix_latest']):
-            if path.startswith(k) and signing_cert_name != v:
-                log.critical("Munged low-security tree scopes trying to access"
-                             " nightly/release buckets for beetmover")
-                sys.exit(3)
+def get_task_cert_type(task):
```

Comment:
Yeah, I striked-through the old scopes validation function and borrowed these both functions from signing script as they were clearer in my opinion. I'll tweak to a more beetmover-type verbiage, thanks for the heads-up.

I'm a little bit confused by the second part. Might be the Friday night so I'll reread this tomorrow morning with fresh eyes. But I'm not getting which scope check you're talking about:

a) hm, let me try this again - you mean that in the future future, you could trigger behavior/action such as "push-go-nightly' and "push-to-candidates" and "push-to-releases" from *try* (hence with "dep" scope),hence allowing developers test a "nightly" graph, a "candidate" graph or a "release" graph with the constraint that all these go under staging-bucket/pub/{firefox,mobile}/{nightly,candidates,releasess}? 

b) the credentials/bucket scope you are asking for - are you talking about bringing back the check I just deleted, the one with: 

`-RESTRICTED_BUCKET_PATHS = {
-    'pub/mobile/nightly': 'nightly',
-    'pub/firefox/nightly': 'nightly',
-    'pub/mobile/release': 'release',
-    'pub/firefox/release': 'release',`

?